### DISCUSSION TITLE
[ProtonMail - remove forced signature] Remove only ProtonMail signature without removing user signature?

### DISCUSSION THREAD

**Discussion by rohandebsarkar** [darkred/Userscripts#49] [Q&A] [answered]
I would like to remove only ProtonMail (class: `protonmail_signature_block-proton`) signature without removing the whole signature block, so that I can keep the user signature (class: `protonmail_signature_block-user`) block.

**Comment by darkred** [comment, marked as answer]
A lot of things have changed in the site code after the recent layout update.
Please confirm that it's working ok for you in the new version.

**Reply by rohandebsarkar** [reply]
Thank you for replying. Yes, the new version after commit f47e8b2b5adebfe2bf3f54928e7218b72ebfefc3 works smoothly for me.

**Reply by rohandebsarkar** [reply]
And also I noticed, it is written in the README that this script does not work for Firefox and also with Greasemonkey. But in my experience it works with both of them. I am currently using this script on Firefox 101.0 with Greasemonkey version 4.11, and it is completely working fine.

And thanks again for the script.

**Reply by darkred** [reply]
I'm glad, you're welcome.


&nbsp;
Regarding "*Tampermonkey and Violentmonkey are supported - Greasemonkey is not supported*", I mention that because I am not interested in supporting Greasemonkey anymore: in GM4 its API changed (so many things changed) and made it not backwards compatible, so I don't think it's worth the effort. I mention the reason in the main README page:

> Greasemonkey -any version- is not supported (the scripts might work, but are not tested).
